docs/domains/Domain.json

```diff
@@ -270,6 +270,10 @@
           "description": "The in-pod name of the directory in which to store the domain, node manager, server logs, and server  *.out files",
           "type": "string"
         },
+        "failBootOnSituationalError": {
+          "description": "If true (the default), on WebLogic server that supports this feature, the WebLogic server  boot would fail if any errors occur when applying situational configuration  during server startup. If false, WebLogic server would start if there are errors in the situational configuration files, but some configuration overrides may be skipped.",
+          "type": "boolean"
+        },
```

Comment:
Would 'failBootOnSituationalConfigurationError' be too long?  I think the official name of teh WL sit-cfg feature is 'situational configuration'.

Reply:
I had originally suggested we just document an environment variable customers could specify instead of adding another Domain attribute.   Now that you've already done the work that's fine IMO, but, OTOH I still have my doubts if we should be cluttering the main domain spec with this parameter.   Up to you.  Or ask Ryan to weigh in...

Reply:
@rjeberhard Would like your input on whether we should add a Domain attribute here or not. Thanks.

Reply:
Agree with your concern. I decided to remove the Domain spec attribute and just go with the env variable.